src/bin/pg_autoctl/pgctl.c

```diff
@@ -271,14 +267,15 @@ pg_include_config(const char *configFilePath,
 	}
 
 	/* find the include 'postgresql-auto-failover.conf' line */
-	includeLine = regexp_first_match(currentConfContents, configIncludeRegex);
-	if (includeLine != NULL)
+	includeLine = strstr(currentConfContents, configIncludeLine);
+
+	if (includeLine != NULL && (includeLine ==  currentConfContents ||
+								includeLine[-1] == '\n'))
```

Comment:
includeLine[-1]  is not valid if `includeLine ==  currentConfContents` whiich is also covered here. some comment would be nice

Reply:
That's a copy/paste from our file `pghba.c` where there was no specific comment, so I though it would be ok...